ironfish-cli/src/commands/chain/import.ts

```diff
@@ -123,17 +122,17 @@ export default class ImportSnapshot extends IronfishCommand {
     CliUx.ux.action.start(`Unzipping ${snapshotPath}`)
     await this.unzip(snapshotPath, tempDir)
     CliUx.ux.action.stop('...done')
-    const blockExportPath = this.sdk.fileSystem.join(tempDir, 'blocks')
 
-    const files = await fsAsync.readdir(blockExportPath)
-    files.sort((a, b) => Number(a) - Number(b))
+    const databaseName = this.sdk.config.get('databaseName')
 
-    const client = await this.sdk.connectRpc()
+    const snapshotDatabasePath = this.sdk.fileSystem.join(tempDir, databaseName)
+    const chainDatabasePath = this.sdk.fileSystem.resolve(this.sdk.config.chainDatabasePath)
 
-    for (const file of files) {
-      const blocks = await fsAsync.readFile(path.join(blockExportPath, file))
-      await client.importSnapshot({ blocks })
-    }
+    CliUx.ux.action.start(
+      `Copying snapshot from ${snapshotDatabasePath} to ${chainDatabasePath}`,
+    )
+    await fsAsync.cp(snapshotDatabasePath, chainDatabasePath, { recursive: true, force: true })
```

Comment:
Should we do a move instead of a copy? That way only the disk inodes need to change and not to copy over 30 GB or more of files?

Reply:
Yeah a move would be better -- faster to rename and doesn't leave an extra 30GB sitting around